client/src/components/LevelLayout.tsx

```diff
@@ -50,6 +56,19 @@ const LevelLayout: React.FC<Props> = ({
         {showInput && <StepValidation />}
         {children}
       </Container>
+      <Flex>
+        <Spacer />
+        <Box
```

Comment:
I think the conditional rendering should be around the entire `<Box />` component, not only the text. This will prevent the little gray square from rendering on level 1.